styles/Layout.module.css

```diff
@@ -0,0 +1,36 @@
+.header {
+  padding: 1rem;
+  display: flex;
+  flex-direction: row;
+  align-items: flex-end;
+  justify-content: space-between;
+}
+
+.navigation {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+}
+
+.navigation > li {
+  margin-left: 1.5rem;
+}
+
+.navitem {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-size: 1rem;
+}
+
+.main {
```

Comment:
Out of curiosity - why use a class instead of just the main element here? There shouldn't be multiples, right? Or is this a css modules convension?

Reply:
Yeah, it should be like that. But yeah, this is a css-modules thing - when I tried to use the main element, I got some errors about not using a local class 🙄